src/voice.ts

```diff
@@ -28,12 +22,15 @@ export enum VoiceMode {
   FULL = 3,
 }
 
+/**
+ * Voice class is mainly a container
+ * object to group `Tickables` for formatting.
+ */
```

Comment:
How about:

\`Voice\` is mainly a container object to group \`Tickables\` for formatting.

Reply:
Done